tests/NuGet.Services.AzureSearch.Tests/AuxiliaryFiles/OwnerDataClientFacts.cs

```diff
@@ -41,6 +41,12 @@ public async Task AllowsEmptyObject()
 
                 Assert.Empty(output.Result);
                 Assert.Equal(ETag, output.AccessCondition.IfMatchETag);
+
```

Comment:
I didn't change the owner data client. The new assertions cover existing behavior